modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -6,23 +6,23 @@ endif::[]
 
 
 [[short_description]]
-The examples here provide guidance on setting up a Gmail connection using Email Connector over the available protocols.
+These examples provide guidance for setting up a Gmail connection using Email Connector over the available protocols.
 
 == Gmail Configurations
 
-By default, Google might block third-party mail clients. For Google to authorize a third-party server to access your account, you must enable `Less Secure Apps` on your Gmail account. You can read a full explanation of the risks involved and get a full explanation https://support.google.com/accounts/answer/6010255[here], which essentially says Google blocks any apps that are not using security protocols that Google deems mandatory unless they allow "less secure apps" to access your Gmail account.
+By default, Google might block third-party mail clients. For Google to authorize a third-party server to access your account, you must enable the *Less Secure Apps* setting for your Gmail account. 
 
-So when you try to connect for the first time, you will receive an email that contains a link for enabling Less Secure Apps in your account.
+You can read a full explanation of the risks involved in enabling less secure apps and get a full explanation https://support.google.com/accounts/answer/6010255[here]. Essentially, Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
 
-image::email/email-gmail-security.jpg[Email notification from Google about the blocked sign-in attempt,height=389,width=421]
+When you try to connect to your Gmail for the first time, you will receive an email that contains a link for enabling less secure apps in your Gmail account.
```

Comment:
Should this sentence PRECEDE the one that I am suggesting replace all of the previous text?

```suggestion
When you try to connect to your Gmail account for the first time, you receive an email that contains a link for enabling less-secure apps in your Gmail account:
+ image::email/email-gmail-security.jpg[Email notification from Google about the blocked sign-in attempt,height=389,width=421]
+ If you do not use the link, Google blocks any apps that are not using security protocols that Google deems mandatory.
```